.husky/pre-commit

```diff
@@ -1 +1 @@
-npx pretty-quick --staged
+npx pretty-quick --staged --ignore-path .prettierignore
```

Comment:
@ullasholla: Does `pnpx` work here?

Reply:
No, didn't work for me. It couldn't find `pnpx` or `pnpm` 🤷‍♂️ 
That's the first thing I tried. Does it work for you?

Reply:
I must have had a typo before, just `pnpm` works. Thanks @kranthie-sap